extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -0,0 +1,46 @@
+define('AuthModal', ['jquery', 'AuthComponent', 'wikia.document'], function ($, AuthComponent) {
+	'use strict';
+
+	var modal,
+		$blackout;
+
+	function open () {
+		$('.WikiaSiteWrapper').append(
+			'<div class="auth-blackout"><div class="auth-modal loading"><span></span><a class="close"></div></div>'
```

Comment:
Did you consider using ```::after``` or ```::before``` pseudoelements instead of introducing empty HTML tag, that has no meaning.